test/e2e/framework/metrics_util.go

```diff
@@ -130,6 +131,8 @@ func (m *MetricsForE2E) SummaryKind() string {
 	return "MetricsForE2E"
 }
 
+var SchedulingLatencyMetricName = model.LabelValue(schedulerMetric.SchedulerSubsystem + "_" + schedulerMetric.SchedulingLatencyName)
```

Comment:
Maybe it'll be cleaner to have sth similar to apiserver like this:

```
var InterestingSchedulerMetrics = []string{
  "scheduler_whatever"
}
```

Reply:
The problem is that scheduler latency metric has a special handling (other metrics will have different structure/labels). Always at some point you will need to verify if name == SCHEDULER_METRIC_NAME. That's why I prefer having variable to having value in array.

Reply:
Yes. However, isn't it the same for apiserver metrics too?

Reply:
I don't think so. The apiserver e2e metric are group of metrics with the some structure, so none of those metric requires special handling.
The apiserver latency metrics are more similar to scheduler latency metric. However in this case, metric names are compared to explicit strings.